doc/Tribler development on Windows.md

```diff
@@ -0,0 +1,101 @@
+This page contains information about setting up a Tribler development environment on Windows. Unlike Linux based systems where installing third-party libraries is often a single `apt-get` command, installing and configuring the necessary libraries requires more attention on Windows. Moreover, the Windows environment has different file stuctures. For instance, where Linux is working extensively with .so (shared object) files, Windows uses DLL files.
+
+## Introduction
+In this guide, all required dependencies of Tribler will be explained. We will discuss how to install these dependencies. Some dependencies have to be built from source whereas other dependencies can be installed using a .msi installer. The guide targets Windows 10, 64-bit systems, however, it is probably not very hard to install 32-bit packages.
+
+First, Python 2.7 should be installed. Python can be downloaded from [their website](https://www.python.org/downloads/release/python-2710/). You should download the Windows x86-64 MSI Installer which is an executable. During the installation, remember to install pip/setuptools and to add Python to the PATH variable to access python from the command line. You can verify whether Python is installed correctly by typing `python` in the command line.
+
+If you did not change the default installation location, Python should be located at `C:\Python27\`. The third-party libraries are located in `C:\Python27\Lib\site-packages`.
+
+In order to compile some of the dependencies of Tribler, Visual Studio 2015 is required which can be downloaded from [here](https://www.visualstudio.com/downloads/download-visual-studio-vs). You should select the community edition. Visual Studio ships with a command line interface that can be used for building some of the Python packages. Moreover, it provides a nice IDE which can be used to work on Python projects.
```

Comment:
We discussed this on a separate channel, add that it must be downloaded (see http://stackoverflow.com/questions/22290501/can-i-download-the-visual-c-command-line-compiler-without-visual-studio)